plugins/components/plugin-area/index.js

```diff
@@ -56,12 +56,12 @@ class PluginArea extends Component {
 					const { render: Plugin } = plugin;
 
 					return (
-						<PluginContextProvider
+						<PluginContext.Provider
 							key={ plugin.name }
-							pluginName={ plugin.name }
+							value={ { pluginName: plugin.name } }
```

Comment:
How often does this re-render?

There's a concern that we're creating a new object here, and since `{} !== {}` (new objects are never referentially equal), React will be forced to reconcile.

Should we just pass `plugin` verbatim as the value?